### ISSUE TITLE
Application error 

### ISSUE THREAD

**Issue by EthraZa** [lyc8503/UptimeFlare#52]
Hi!

I have followed the Quickstart guide nad everything is green, but now it is just showing an error that I have no clue on how to debug or fix:

buld-and-deploy is green, but clicking on it I see this red stuff on Deploy using Terraform:
```
cloudflare_worker_cron_trigger.uptimeflare_worker_cron: Creating...
cloudflare_pages_project.uptimeflare: Creation complete after 7s [id=uptimeflare]
╷
│ Error: failed to update Worker Cron Trigger: workers.api.error.subdomain_required (10063)
│ 
│   with cloudflare_worker_cron_trigger.uptimeflare_worker_cron,
│   on deploy.tf line 37, in resource "cloudflare_worker_cron_trigger" "uptimeflare_worker_cron":
│   37: resource "cloudflare_worker_cron_trigger" "uptimeflare_worker_cron" {
│ 
╵
Error: Terraform exited with code 1.
Error: Process completed with exit code 1.
```

Rebuild with debug on:
```
##[debug]Plan: 4 to add, 0 to change, 0 to destroy.
##[debug]cloudflare_workers_kv_namespace.uptimeflare_kv: Creating...
##[debug]
##[debug]stderr: ╷
##[debug]│ Error: error creating workers kv namespace: create namespace: 'a namespace with this account ID and title already exists' (10014)
##[debug]│ 
##[debug]│   with cloudflare_workers_kv_namespace.uptimeflare_kv,
##[debug]│   on deploy.tf line 19, in resource "cloudflare_workers_kv_namespace" "uptimeflare_kv":
##[debug]│   19: resource "cloudflare_workers_kv_namespace" "uptimeflare_kv" {
##[debug]│ 
##[debug]╵
##[debug]
##[debug]exitcode: 1
Error: Terraform exited with code 1.
Error: Process completed with exit code 1.
```

Deleted everything in Cloudflare and try to redeploy at Github, now it seens worse:
```
Run npx wrangler pages deploy .vercel/output/static --project-name uptimeflare

✘ [ERROR] A request to the Cloudflare API (/accounts/0dd3a062b7d4f3233f97[8](https://github.com/GHSix/UptimeFlare/actions/runs/10014296488/job/27683728753#step:12:9)4b5a144f2[9](https://github.com/GHSix/UptimeFlare/actions/runs/10014296488/job/27683728753#step:12:10)1/pages/projects/uptimeflare) failed.

  Project not found. The specified project name does not match any of your existing projects. [code: 8000007]
  
  If you think this is a bug, please open an issue at: https://github.com/cloudflare/workers-sdk/issues/new/choose


Error: Process completed with exit code 1.
```

On site:

`Application error: a client-side exception has occurred (see the browser console for more information).`

On console:

```
Navigated to https://5f2cb483.uptimeflare-55y.pages.dev/
_app-f7bfd027cc9448b7.js:23 TypeError: Cannot read properties of null (reading 'overallUp')
    at OverallStatus (index-807a2a7b5f3cfc68.js:1:731)
    at Xh (framework-0c7baedefba6b077.js:9:60990)
    at x (framework-0c7baedefba6b077.js:9:119445)
    at Vk (framework-0c7baedefba6b077.js:9:99130)
    at framework-0c7baedefba6b077.js:9:98997
    at Jk (framework-0c7baedefba6b077.js:9:99004)
    at Ok (framework-0c7baedefba6b077.js:9:95760)
    at Fk (framework-0c7baedefba6b077.js:9:96149)
    at jg (framework-0c7baedefba6b077.js:9:44899)
    at Ik (framework-0c7baedefba6b077.js:9:113314)
console.error @ _app-f7bfd027cc9448b7.js:23
Mi @ framework-0c7baedefba6b077.js:9
Ri.o.componentDidCatch.a.callback @ framework-0c7baedefba6b077.js:9
ih @ framework-0c7baedefba6b077.js:9
lk @ framework-0c7baedefba6b077.js:9
jk @ framework-0c7baedefba6b077.js:9
(anonymous) @ framework-0c7baedefba6b077.js:9
Qk @ framework-0c7baedefba6b077.js:9
Fk @ framework-0c7baedefba6b077.js:9
jg @ framework-0c7baedefba6b077.js:9
Ik @ framework-0c7baedefba6b077.js:9
w @ framework-0c7baedefba6b077.js:9
J @ framework-0c7baedefba6b077.js:33
R @ framework-0c7baedefba6b077.js:33
_app-f7bfd027cc9448b7.js:23 TypeError: Cannot read properties of null (reading 'overallUp')
    at OverallStatus (index-807a2a7b5f3cfc68.js:1:731)
    at Xh (framework-0c7baedefba6b077.js:9:60990)
    at x (framework-0c7baedefba6b077.js:9:119445)
    at Vk (framework-0c7baedefba6b077.js:9:99130)
    at framework-0c7baedefba6b077.js:9:98997
    at Jk (framework-0c7baedefba6b077.js:9:99004)
    at Ok (framework-0c7baedefba6b077.js:9:95760)
    at Fk (framework-0c7baedefba6b077.js:9:96149)
    at jg (framework-0c7baedefba6b077.js:9:44899)
    at Ik (framework-0c7baedefba6b077.js:9:113314)
console.error @ _app-f7bfd027cc9448b7.js:23
renderError @ main-a82dbdedafde9961.js:1
fn @ main-a82dbdedafde9961.js:1
componentDidCatch @ main-a82dbdedafde9961.js:1
Ri.o.componentDidCatch.a.callback @ framework-0c7baedefba6b077.js:9
ih @ framework-0c7baedefba6b077.js:9
lk @ framework-0c7baedefba6b077.js:9
jk @ framework-0c7baedefba6b077.js:9
(anonymous) @ framework-0c7baedefba6b077.js:9
Qk @ framework-0c7baedefba6b077.js:9
Fk @ framework-0c7baedefba6b077.js:9
jg @ framework-0c7baedefba6b077.js:9
Ik @ framework-0c7baedefba6b077.js:9
w @ framework-0c7baedefba6b077.js:9
J @ framework-0c7baedefba6b077.js:33
R @ framework-0c7baedefba6b077.js:33
_app-f7bfd027cc9448b7.js:23 A client-side exception has occurred, see here for more info: https://nextjs.org/docs/messages/client-side-exception-occurred
console.error @ _app-f7bfd027cc9448b7.js:23
renderError @ main-a82dbdedafde9961.js:1
fn @ main-a82dbdedafde9961.js:1
componentDidCatch @ main-a82dbdedafde9961.js:1
Ri.o.componentDidCatch.a.callback @ framework-0c7baedefba6b077.js:9
ih @ framework-0c7baedefba6b077.js:9
lk @ framework-0c7baedefba6b077.js:9
jk @ framework-0c7baedefba6b077.js:9
(anonymous) @ framework-0c7baedefba6b077.js:9
Qk @ framework-0c7baedefba6b077.js:9
Fk @ framework-0c7baedefba6b077.js:9
jg @ framework-0c7baedefba6b077.js:9
Ik @ framework-0c7baedefba6b077.js:9
w @ framework-0c7baedefba6b077.js:9
J @ framework-0c7baedefba6b077.js:33
R @ framework-0c7baedefba6b077.js:33
_app-f7bfd027cc9448b7.js:23 TypeError: Cannot read properties of null (reading 'latency')
    at MonitorDetail (index-807a2a7b5f3cfc68.js:1:123373)
    at Xh (framework-0c7baedefba6b077.js:9:60990)
    at x (framework-0c7baedefba6b077.js:9:119445)
    at Vk (framework-0c7baedefba6b077.js:9:99130)
    at framework-0c7baedefba6b077.js:9:98997
    at Jk (framework-0c7baedefba6b077.js:9:99004)
    at Ok (framework-0c7baedefba6b077.js:9:95760)
    at Fk (framework-0c7baedefba6b077.js:9:96149)
    at jg (framework-0c7baedefba6b077.js:9:44899)
    at Ik (framework-0c7baedefba6b077.js:9:113314)
console.error @ _app-f7bfd027cc9448b7.js:23
Mi @ framework-0c7baedefba6b077.js:9
Ri.o.componentDidCatch.a.callback @ framework-0c7baedefba6b077.js:9
ih @ framework-0c7baedefba6b077.js:9
lk @ framework-0c7baedefba6b077.js:9
jk @ framework-0c7baedefba6b077.js:9
(anonymous) @ framework-0c7baedefba6b077.js:9
Qk @ framework-0c7baedefba6b077.js:9
Fk @ framework-0c7baedefba6b077.js:9
jg @ framework-0c7baedefba6b077.js:9
Ik @ framework-0c7baedefba6b077.js:9
w @ framework-0c7baedefba6b077.js:9
J @ framework-0c7baedefba6b077.js:33
R @ framework-0c7baedefba6b077.js:33
_app-f7bfd027cc9448b7.js:23 TypeError: Cannot read properties of null (reading 'latency')
    at MonitorDetail (index-807a2a7b5f3cfc68.js:1:123373)
    at Xh (framework-0c7baedefba6b077.js:9:60990)
    at x (framework-0c7baedefba6b077.js:9:119445)
    at Vk (framework-0c7baedefba6b077.js:9:99130)
    at framework-0c7baedefba6b077.js:9:98997
    at Jk (framework-0c7baedefba6b077.js:9:99004)
    at Ok (framework-0c7baedefba6b077.js:9:95760)
    at Fk (framework-0c7baedefba6b077.js:9:96149)
    at jg (framework-0c7baedefba6b077.js:9:44899)
    at Ik (framework-0c7baedefba6b077.js:9:113314)
console.error @ _app-f7bfd027cc9448b7.js:23
renderError @ main-a82dbdedafde9961.js:1
fn @ main-a82dbdedafde9961.js:1
componentDidCatch @ main-a82dbdedafde9961.js:1
Ri.o.componentDidCatch.a.callback @ framework-0c7baedefba6b077.js:9
ih @ framework-0c7baedefba6b077.js:9
lk @ framework-0c7baedefba6b077.js:9
jk @ framework-0c7baedefba6b077.js:9
(anonymous) @ framework-0c7baedefba6b077.js:9
Qk @ framework-0c7baedefba6b077.js:9
Fk @ framework-0c7baedefba6b077.js:9
jg @ framework-0c7baedefba6b077.js:9
Ik @ framework-0c7baedefba6b077.js:9
w @ framework-0c7baedefba6b077.js:9
J @ framework-0c7baedefba6b077.js:33
R @ framework-0c7baedefba6b077.js:33
_app-f7bfd027cc9448b7.js:23 A client-side exception has occurred, see here for more info: https://nextjs.org/docs/messages/client-side-exception-occurred
console.error @ _app-f7bfd027cc9448b7.js:23
renderError @ main-a82dbdedafde9961.js:1
fn @ main-a82dbdedafde9961.js:1
componentDidCatch @ main-a82dbdedafde9961.js:1
Ri.o.componentDidCatch.a.callback @ framework-0c7baedefba6b077.js:9
ih @ framework-0c7baedefba6b077.js:9
lk @ framework-0c7baedefba6b077.js:9
jk @ framework-0c7baedefba6b077.js:9
(anonymous) @ framework-0c7baedefba6b077.js:9
Qk @ framework-0c7baedefba6b077.js:9
Fk @ framework-0c7baedefba6b077.js:9
jg @ framework-0c7baedefba6b077.js:9
Ik @ framework-0c7baedefba6b077.js:9
w @ framework-0c7baedefba6b077.js:9
J @ framework-0c7baedefba6b077.js:33
R @ framework-0c7baedefba6b077.js:33
_app-f7bfd027cc9448b7.js:23 TypeError: Cannot read properties of null (reading 'latency')
    at MonitorDetail (index-807a2a7b5f3cfc68.js:1:123373)
    at Xh (framework-0c7baedefba6b077.js:9:60990)
    at x (framework-0c7baedefba6b077.js:9:119445)
    at Vk (framework-0c7baedefba6b077.js:9:99130)
    at framework-0c7baedefba6b077.js:9:98997
    at Jk (framework-0c7baedefba6b077.js:9:99004)
    at Ok (framework-0c7baedefba6b077.js:9:95760)
    at Fk (framework-0c7baedefba6b077.js:9:96149)
    at jg (framework-0c7baedefba6b077.js:9:44899)
    at Ik (framework-0c7baedefba6b077.js:9:113314)
console.error @ _app-f7bfd027cc9448b7.js:23
Mi @ framework-0c7baedefba6b077.js:9
Ri.o.componentDidCatch.a.callback @ framework-0c7baedefba6b077.js:9
ih @ framework-0c7baedefba6b077.js:9
lk @ framework-0c7baedefba6b077.js:9
jk @ framework-0c7baedefba6b077.js:9
(anonymous) @ framework-0c7baedefba6b077.js:9
Qk @ framework-0c7baedefba6b077.js:9
Fk @ framework-0c7baedefba6b077.js:9
jg @ framework-0c7baedefba6b077.js:9
Ik @ framework-0c7baedefba6b077.js:9
w @ framework-0c7baedefba6b077.js:9
J @ framework-0c7baedefba6b077.js:33
R @ framework-0c7baedefba6b077.js:33
_app-f7bfd027cc9448b7.js:23 TypeError: Cannot read properties of null (reading 'latency')
    at MonitorDetail (index-807a2a7b5f3cfc68.js:1:123373)
    at Xh (framework-0c7baedefba6b077.js:9:60990)
    at x (framework-0c7baedefba6b077.js:9:119445)
    at Vk (framework-0c7baedefba6b077.js:9:99130)
    at framework-0c7baedefba6b077.js:9:98997
    at Jk (framework-0c7baedefba6b077.js:9:99004)
    at Ok (framework-0c7baedefba6b077.js:9:95760)
    at Fk (framework-0c7baedefba6b077.js:9:96149)
    at jg (framework-0c7baedefba6b077.js:9:44899)
    at Ik (framework-0c7baedefba6b077.js:9:113314)
console.error @ _app-f7bfd027cc9448b7.js:23
renderError @ main-a82dbdedafde9961.js:1
fn @ main-a82dbdedafde9961.js:1
componentDidCatch @ main-a82dbdedafde9961.js:1
Ri.o.componentDidCatch.a.callback @ framework-0c7baedefba6b077.js:9
ih @ framework-0c7baedefba6b077.js:9
lk @ framework-0c7baedefba6b077.js:9
jk @ framework-0c7baedefba6b077.js:9
(anonymous) @ framework-0c7baedefba6b077.js:9
Qk @ framework-0c7baedefba6b077.js:9
Fk @ framework-0c7baedefba6b077.js:9
jg @ framework-0c7baedefba6b077.js:9
Ik @ framework-0c7baedefba6b077.js:9
w @ framework-0c7baedefba6b077.js:9
J @ framework-0c7baedefba6b077.js:33
R @ framework-0c7baedefba6b077.js:33
_app-f7bfd027cc9448b7.js:23 A client-side exception has occurred, see here for more info: https://nextjs.org/docs/messages/client-side-exception-occurred
console.error @ _app-f7bfd027cc9448b7.js:23
renderError @ main-a82dbdedafde9961.js:1
fn @ main-a82dbdedafde9961.js:1
componentDidCatch @ main-a82dbdedafde9961.js:1
Ri.o.componentDidCatch.a.callback @ framework-0c7baedefba6b077.js:9
ih @ framework-0c7baedefba6b077.js:9
lk @ framework-0c7baedefba6b077.js:9
jk @ framework-0c7baedefba6b077.js:9
(anonymous) @ framework-0c7baedefba6b077.js:9
Qk @ framework-0c7baedefba6b077.js:9
Fk @ framework-0c7baedefba6b077.js:9
jg @ framework-0c7baedefba6b077.js:9
Ik @ framework-0c7baedefba6b077.js:9
w @ framework-0c7baedefba6b077.js:9
J @ framework-0c7baedefba6b077.js:33
R @ framework-0c7baedefba6b077.js:33
_app-f7bfd027cc9448b7.js:23 TypeError: Cannot read properties of null (reading 'latency')
    at MonitorDetail (index-807a2a7b5f3cfc68.js:1:123373)
    at Xh (framework-0c7baedefba6b077.js:9:60990)
    at x (framework-0c7baedefba6b077.js:9:119445)
    at Vk (framework-0c7baedefba6b077.js:9:99130)
    at framework-0c7baedefba6b077.js:9:98997
    at Jk (framework-0c7baedefba6b077.js:9:99004)
    at Ok (framework-0c7baedefba6b077.js:9:95760)
    at Fk (framework-0c7baedefba6b077.js:9:96149)
    at jg (framework-0c7baedefba6b077.js:9:44899)
    at Ik (framework-0c7baedefba6b077.js:9:113314)
console.error @ _app-f7bfd027cc9448b7.js:23
Mi @ framework-0c7baedefba6b077.js:9
Ri.o.componentDidCatch.a.callback @ framework-0c7baedefba6b077.js:9
ih @ framework-0c7baedefba6b077.js:9
lk @ framework-0c7baedefba6b077.js:9
jk @ framework-0c7baedefba6b077.js:9
(anonymous) @ framework-0c7baedefba6b077.js:9
Qk @ framework-0c7baedefba6b077.js:9
Fk @ framework-0c7baedefba6b077.js:9
jg @ framework-0c7baedefba6b077.js:9
Ik @ framework-0c7baedefba6b077.js:9
w @ framework-0c7baedefba6b077.js:9
J @ framework-0c7baedefba6b077.js:33
R @ framework-0c7baedefba6b077.js:33
_app-f7bfd027cc9448b7.js:23 TypeError: Cannot read properties of null (reading 'latency')
    at MonitorDetail (index-807a2a7b5f3cfc68.js:1:123373)
    at Xh (framework-0c7baedefba6b077.js:9:60990)
    at x (framework-0c7baedefba6b077.js:9:119445)
    at Vk (framework-0c7baedefba6b077.js:9:99130)
    at framework-0c7baedefba6b077.js:9:98997
    at Jk (framework-0c7baedefba6b077.js:9:99004)
    at Ok (framework-0c7baedefba6b077.js:9:95760)
    at Fk (framework-0c7baedefba6b077.js:9:96149)
    at jg (framework-0c7baedefba6b077.js:9:44899)
    at Ik (framework-0c7baedefba6b077.js:9:113314)
console.error @ _app-f7bfd027cc9448b7.js:23
renderError @ main-a82dbdedafde9961.js:1
fn @ main-a82dbdedafde9961.js:1
componentDidCatch @ main-a82dbdedafde9961.js:1
Ri.o.componentDidCatch.a.callback @ framework-0c7baedefba6b077.js:9
ih @ framework-0c7baedefba6b077.js:9
lk @ framework-0c7baedefba6b077.js:9
jk @ framework-0c7baedefba6b077.js:9
(anonymous) @ framework-0c7baedefba6b077.js:9
Qk @ framework-0c7baedefba6b077.js:9
Fk @ framework-0c7baedefba6b077.js:9
jg @ framework-0c7baedefba6b077.js:9
Ik @ framework-0c7baedefba6b077.js:9
w @ framework-0c7baedefba6b077.js:9
J @ framework-0c7baedefba6b077.js:33
R @ framework-0c7baedefba6b077.js:33
_app-f7bfd027cc9448b7.js:23 A client-side exception has occurred, see here for more info: https://nextjs.org/docs/messages/client-side-exception-occurred
console.error @ _app-f7bfd027cc9448b7.js:23
renderError @ main-a82dbdedafde9961.js:1
fn @ main-a82dbdedafde9961.js:1
componentDidCatch @ main-a82dbdedafde9961.js:1
Ri.o.componentDidCatch.a.callback @ framework-0c7baedefba6b077.js:9
ih @ framework-0c7baedefba6b077.js:9
lk @ framework-0c7baedefba6b077.js:9
jk @ framework-0c7baedefba6b077.js:9
(anonymous) @ framework-0c7baedefba6b077.js:9
Qk @ framework-0c7baedefba6b077.js:9
Fk @ framework-0c7baedefba6b077.js:9
jg @ framework-0c7baedefba6b077.js:9
Ik @ framework-0c7baedefba6b077.js:9
w @ framework-0c7baedefba6b077.js:9
J @ framework-0c7baedefba6b077.js:33
R @ framework-0c7baedefba6b077.js:33
_app-f7bfd027cc9448b7.js:23 TypeError: Cannot read properties of null (reading 'latency')
    at MonitorDetail (index-807a2a7b5f3cfc68.js:1:123373)
    at Xh (framework-0c7baedefba6b077.js:9:60990)
    at x (framework-0c7baedefba6b077.js:9:119445)
    at Vk (framework-0c7baedefba6b077.js:9:99130)
    at framework-0c7baedefba6b077.js:9:98997
    at Jk (framework-0c7baedefba6b077.js:9:99004)
    at Ok (framework-0c7baedefba6b077.js:9:95760)
    at Fk (framework-0c7baedefba6b077.js:9:96149)
    at jg (framework-0c7baedefba6b077.js:9:44899)
    at Ik (framework-0c7baedefba6b077.js:9:113314)
console.error @ _app-f7bfd027cc9448b7.js:23
Mi @ framework-0c7baedefba6b077.js:9
Ri.o.componentDidCatch.a.callback @ framework-0c7baedefba6b077.js:9
ih @ framework-0c7baedefba6b077.js:9
lk @ framework-0c7baedefba6b077.js:9
jk @ framework-0c7baedefba6b077.js:9
(anonymous) @ framework-0c7baedefba6b077.js:9
Qk @ framework-0c7baedefba6b077.js:9
Fk @ framework-0c7baedefba6b077.js:9
jg @ framework-0c7baedefba6b077.js:9
Ik @ framework-0c7baedefba6b077.js:9
w @ framework-0c7baedefba6b077.js:9
J @ framework-0c7baedefba6b077.js:33
R @ framework-0c7baedefba6b077.js:33
_app-f7bfd027cc9448b7.js:23 TypeError: Cannot read properties of null (reading 'latency')
    at MonitorDetail (index-807a2a7b5f3cfc68.js:1:123373)
    at Xh (framework-0c7baedefba6b077.js:9:60990)
    at x (framework-0c7baedefba6b077.js:9:119445)
    at Vk (framework-0c7baedefba6b077.js:9:99130)
    at framework-0c7baedefba6b077.js:9:98997
    at Jk (framework-0c7baedefba6b077.js:9:99004)
    at Ok (framework-0c7baedefba6b077.js:9:95760)
    at Fk (framework-0c7baedefba6b077.js:9:96149)
    at jg (framework-0c7baedefba6b077.js:9:44899)
    at Ik (framework-0c7baedefba6b077.js:9:113314)
console.error @ _app-f7bfd027cc9448b7.js:23
renderError @ main-a82dbdedafde9961.js:1
fn @ main-a82dbdedafde9961.js:1
componentDidCatch @ main-a82dbdedafde9961.js:1
Ri.o.componentDidCatch.a.callback @ framework-0c7baedefba6b077.js:9
ih @ framework-0c7baedefba6b077.js:9
lk @ framework-0c7baedefba6b077.js:9
jk @ framework-0c7baedefba6b077.js:9
(anonymous) @ framework-0c7baedefba6b077.js:9
Qk @ framework-0c7baedefba6b077.js:9
Fk @ framework-0c7baedefba6b077.js:9
jg @ framework-0c7baedefba6b077.js:9
Ik @ framework-0c7baedefba6b077.js:9
w @ framework-0c7baedefba6b077.js:9
J @ framework-0c7baedefba6b077.js:33
R @ framework-0c7baedefba6b077.js:33
_app-f7bfd027cc9448b7.js:23 A client-side exception has occurred, see here for more info: https://nextjs.org/docs/messages/client-side-exception-occurred
console.error @ _app-f7bfd027cc9448b7.js:23
renderError @ main-a82dbdedafde9961.js:1
fn @ main-a82dbdedafde9961.js:1
componentDidCatch @ main-a82dbdedafde9961.js:1
Ri.o.componentDidCatch.a.callback @ framework-0c7baedefba6b077.js:9
ih @ framework-0c7baedefba6b077.js:9
lk @ framework-0c7baedefba6b077.js:9
jk @ framework-0c7baedefba6b077.js:9
(anonymous) @ framework-0c7baedefba6b077.js:9
Qk @ framework-0c7baedefba6b077.js:9
Fk @ framework-0c7baedefba6b077.js:9
jg @ framework-0c7baedefba6b077.js:9
Ik @ framework-0c7baedefba6b077.js:9
w @ framework-0c7baedefba6b077.js:9
J @ framework-0c7baedefba6b077.js:33
R @ framework-0c7baedefba6b077.js:33
4_app-f7bfd027cc9448b7.js:23 Error rendering page:  Error: Cancel rendering route
    at _ (main-a82dbdedafde9961.js:1:13174)
    at main-a82dbdedafde9961.js:1:13126
    at new Promise (<anonymous>)
    at doRender (main-a82dbdedafde9961.js:1:13103)
    at main-a82dbdedafde9961.js:1:11240
```

**Comment by lyc8503** [comment]
Hello! Thanks for the detailed issue report, I think I know how to fix this.

The `subdomain_required` issue you initially encountered was supposed to have been fixed in d827a70, but for some reason it came back, I'll try to look into it later.

Your subsequent attempts to remove the resource and redeploy it were also correct, the only minor error was that you forgot to remove the KV, which caused the second deployment to fail.

Now, you should open https://dash.cloudflare.com/redirect?account=workers/kv/namespaces and delete one of the KVs associated with UptimeFlare, then open https://dash.cloudflare.com/redirect?account=workers-and-pages, delete the two items starting with UptimeFlare, and retrigger the actions deployment. This should solve your problem.

On Cloudflare Dashboard, if you delete the worker first, KV won't be deleted along with it, but you can't see it on Cloudflare Dashboard either, since opening the KV page will force you to jump to the wizard for creating a new worker. This is quite misleading and seems to be a small front-end bug in Cloudflare itself.

**Comment by lyc8503** [comment]
By the way, are you using a new Cloudflare account? The `subdomain_required` only seems to show up on accounts that haven't used Workers before, but I'm really not quite sure how to reproduce that and how to fix it.

**Comment by EthraZa** [comment]
My CF account was created around 18/07/2020. But yestarday was the first time I had touch workers.

This thing of delete the kv first had bite my ankle. I have redeployed it to be able to delete kv first and deleted workers after. Then I went to redeploy last run with "debug on" to see if it helps, and boom, it just deployed right and is working now.

So the solution to it not installing the first time seens to be: delete everything (kv first), and redeploy.

Thank you for your time and awesome work.
